docs/reference/data-streams/downsampling-manual.asciidoc

```diff
@@ -0,0 +1,450 @@
+
+[source,console]
+----
+POST /sample-01/_downsample/sample-01-downsample
+{
+  "fixed_interval": "1h"
+}
+----
+// TEST[continued]
+
+Finally, delete the original index:
+
+[source,console]
+----
+DELETE /sample-01
+----
+// TEST[continued]
+
+[discrete]
+[[downsampling-manual-view-results]]
+==== View the results
+
+
+Now, re-run your search query:
+
+[source,console]
+----
+GET /sample-01*/_search
+----
+// TEST[continued]
+
+In the query results, notice that the numer of hits has been reduced to only 288 documents. As well, for each time series metric statistical representations have been calculated: `min`, `max`, `sum`, and `value_count`.
+
+```
+  "hits": {
+    "total": {
+      "value": 288,
+      "relation": "eq"
+    },
+    "max_score": 1,
+    "hits": [
+      {
+        "_index": "sample-01-downsample",
+        "_id": "WyHN6N6AwdaJByQWAAABgYNYIYA",
+        "_score": 1,
+        "_source": {
+          "@timestamp": "2022-06-20T23:00:00.000Z",
+          "_doc_count": 81,
+          "kubernetes.host": "gke-apps-0",
+          "kubernetes.namespace": "namespace26",
+          "kubernetes.node": "gke-apps-0-1",
+          "kubernetes.pod": "gke-apps-0-1-0",
+          "kubernetes.container.cpu.usage.nanocores": {
+            "min": 23344,
+            "max": 163408,
+            "sum": 7488985,
+            "value_count": 81
+          },
+          "kubernetes.container.memory.available.bytes": {
+            "min": 167751844,
+            "max": 1182251090,
+            "sum": 58169948901,
+            "value_count": 81
+          },
+          "kubernetes.container.memory.rss.bytes": {
+            "min": 54067,
+            "max": 391987,
+            "sum": 17550215,
+            "value_count": 81
+          },
+          "kubernetes.container.memory.pagefaults": {
+            "min": 69086,
+            "max": 428910,
+            "sum": 20239365,
+            "value_count": 81
+          },
+          "kubernetes.container.memory.workingset.bytes": {
+            "min": 323420,
+            "max": 2279342,
+            "sum": 104233700,
+            "value_count": 81
+          },
+          "kubernetes.container.memory.usage.bytes": {
+            "min": 61401416,
+            "max": 413064069,
+            "sum": 18557182404,
+            "value_count": 81
+          }
+        }
+      },
+...
+```
+
+This example demonstrates how downsampling can dramatically reduce the number of records stored for time series data, within whatever time boundaries you choose. It's also possible to perform downsampling on already downsampled data, to further reduce storage and associated costs, as the time series data ages and the data resolution becomes less critical.
+
+Downsampling is very easily integrated within an ILM policy. To learn more, try the <<downsampling-ilm,Run downsampling with ILM>> example.
+
+////
+[source,console]
+----
+DELETE /sample-01*
+DELETE _data_stream/*
+DELETE _index_template/*
+----
```

Comment:
Not sure this cleaning is required.

Reply:
Nice! I thought all the object we created had to be un-created.
Fixed via [347cd81](https://github.com/elastic/elasticsearch/pull/88571/commits/347cd8124f8d37d3efd19095bb154344ed31ce5c)